hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -110,18 +110,17 @@ import Prelude qualified
 allNodeIds :: [Int]
 allNodeIds = [1 .. 3]
 
--- | Like 'withTempDir', busing a common prefix to keep hydra-cluster logs more
--- easily on CI.
+-- | Like 'withTempDir', but using a common template to archive logs more easily
+-- on CI.
 --
 -- NOTE: The ci-nix.yaml workflow depends on this.
-withClusterTempDir :: MonadIO m => String -> (FilePath -> m a) -> m a
-withClusterTempDir name =
-  withTempDir ("hydra-cluster-e2e-" <> name)
+withClusterTempDir :: MonadIO m => (FilePath -> m a) -> m a
```

Comment:
Why not use this everywhere?

Reply:
Now with the many `withTempDir "hydra-cluster"` we could consider doing that, yes.

Do you want me to use this in all tests of `hydra-cluster` package?